integration-tests/api/__tests__/admin/sales-channels.js

```diff
@@ -85,6 +78,49 @@ describe("sales channels", () => {
     })
   })
 
+
+  describe("POST /admin/sales-channels", () => {
+    beforeEach(async () => {
+      try {
+        await adminSeeder(dbConnection)
+      } catch (e) {
+        console.error(e)
+      }
+    })
+
+    afterEach(async () => {
+      const db = useDb()
+      await db.teardown()
+    })
+
+    it("successfully creates a sales channel", async () => {
+      const api = useApi()
+
+      const newSalesChannel = {
+        name: "sales channel name",
+        description: "sales channel description",
+      }
+
+      const response = await api
+        .post("/admin/sales-channels", newSalesChannel, adminReqConfig)
+        .catch((err) => {
+          console.log(err)
+        })
+
+      expect(response.status).toEqual(200)
+      expect(response.data.sales_channel).toBeTruthy()
+
+      expect(response.data).toMatchSnapshot({
```

Comment:
suggestion(non-blocking): Just for consistency:
```ts
    expect(response.data.sales_channel).toMatchSnapshot({
        id: expect.any(String),
        name: newSalesChannel.name,
        description: newSalesChannel.description,
        is_disabled: false,
        created_at: expect.any(String),
        updated_at: expect.any(String),
      })
    })
```
The subsequent snapshot generated will looks close to the other one that way :) take it or leave it :)